src/app/hooks/use-search-parameters-validation.test.ts

```diff
@@ -218,7 +218,7 @@ describe("useSearchParametersValidation", () => {
 		});
 	});
 
-	it("should throw if sign and no message", async () => {
+	it.skip("should throw if sign and no message", async () => {
```

Comment:
🤔 why are they all skipped

Reply:
@ItsANameToo deeplink for signing is not currently working since there's no page to redirect the user to. That task was already too long, and since this needs a refactor of the deeplink handling to open a modal, I figured it was better to add a separate task for that: https://app.clickup.com/t/86dww5u63